test/ClientTest.cs

```diff
@@ -50,5 +52,26 @@ public void TestAuthenticatedRequest()
             Assert.Throws<InvalidOperationException>(() =>
                 Client.AuthenticatedRequest("info"));
         }
+
+
+        [Test]
+        public void ActingAsTest()
+        {
+            string role = $"{Client.GetAccountName()}:{ResourceKind.user}:foo";
+            string resourceVarUri = $"test:///resources/{TestAccount}?{ResourceKind.variable}";
```

Comment:
It may be helpful to review the API routes available here:
https://github.com/cyberark/conjur/blob/master/config/routes.rb

This URL form doesn't actually exist in the V5 API, so the test itself doesn't verify the client API behavior as well as it could. For listing the variables, for example, I believe what you want is:
```
string resourceVarUri = $"test:///resources/{TestAccount}/{ResourceKind.variable}";
```

Reply:
Hi @DvirCyberArk, can you clarify what the current form is intended to request? It looks like it's trying to demonstrate returning the variable resources from Conjur.

When you run a local Conjur server and send a request like:
```
GET https://conjur-test/resources/test-account/variable
```
This will return the resources loaded, filtered to only show the variable types.

However, if you request:
```
GET https://conjur-test/resources/test-account?variable
```
This will return the resources, without any filter applied. The request won't return an error, but the `?variable` parameter is essentially ignored.

I know this is all mocked out for the test, but the tests provide a source of documentation for the API, and so it's important to correctly demonstrate how the API interaction works.